OData/src/System.Web.OData/OData/Query/Expressions/FilterBinder.cs

```diff
@@ -887,6 +890,17 @@ private Expression BindDate(SingleValueFunctionCallNode node)
             return arguments[0];
         }
 
+        private Expression BindNow(SingleValueFunctionCallNode node)
+        {
+            Contract.Assert("now" == node.Name);
+
+            // Function Now() does not take any arguemnts.
+            Expression[] arguments = BindArguments(node.Parameters);
+            Contract.Assert(arguments.Length == 0);
+
+            return Expression.Property(null, typeof(DateTimeOffset), "Now");
```

Comment:
Why not UtcNow instead of Now?  The spec does allow us to use any time zone (but it's example is Utc) but Utc seems to isolate us form the time zone of the server. 

From the spec:
5.1.1.6.9 now
The now function has the following signature:
Edm.DateTimeOffset now()
The now function returns the current point in time (date and time with time zone) as a DateTimeOffset value.
Services are free to choose the time zone for the current point, e.g. UTC. Services that are unable to preserve the offset of Edm.DateTimeOffset values and instead normalize the values to some common time zone SHOULD return a value in the normalized time zone (i.e., UTC).